content/pt/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,387 @@
+---
+title: kubectl Cheat Sheet
+reviewers:
+- erictune
+- krousey
+- clove
+content_template: templates/concept
+card:
+  name: reference
+  weight: 30
+---
+
+{{% capture overview %}}
+
+Veja também: [Visão geral do Kubectl](/docs/reference/kubectl/overview/) e [JsonPath Guide](/docs/reference/kubectl/jsonpath).
+
+Esta página é uma visão geral do comando `kubectl`.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+# kubectl - Cheat Sheet
+
+## Kubectl Autocomplete
+
+### BASH
+
+```bash
+source <(kubectl completion bash) # configuração de autocomplete no bash do shell atual, o pacote bash-completion precisa ter sido instalado primeiro.
+echo "source <(kubectl completion bash)" >> ~/.bashrc # para adicionar o autocomplete permanentemente no seu shell bash.
+```
+
+Você também pode usar um apelido abreviado para `kubectl` que também funciona com o autocomplete:
+
+```bash
+alias k=kubectl
+complete -F __start_kubectl k
+```
+
+### ZSH
+
+```bash
+source <(kubectl completion zsh)  # configuração para usar autocomplete no terminal zsh no shell atual
+echo "if [ $commands[kubectl] ]; then source <(kubectl completion zsh); fi" >> ~/.zshrc # adicionar autocomplete permanentemente para o seu shell zsh
+```
+
+##  Contexto e configuração do Kubectl
+
+Defina com qual cluster do Kubernetes o `kubectl` se comunica e modifique os detales da configuração.
+Veja a documentação [Autenticando entre clusters com o kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) para
+informações detalhadas do arquivo de configuração.
+
+```bash
+kubectl config view # Mostrar configurações do kubeconfig mergeadas.
```

Comment:
```suggestion
kubectl config view # Mostra configurações mescladas do kubeconfig.
```

Reply:
Not sure we should use `mescladas` or `mergeadas`. Honestly I prefer the second one

Reply:
No problem. :)